api/services/ocean/spark/schemas/oceanSparkJob.yaml

```diff
@@ -0,0 +1,55 @@
+type: object
+required:
+- id
+- displayName
+- accountId
+- organizationId
+- clusterId
+- controllerClusterId
+description: >
+  The description of an Ocean Spark job
+properties:
+  internalId:
+    $ref: "internalId.yaml"
+  id:
+    $ref: "oceanSparkJobId.yaml"
+  displayName:
+    $ref: "oceanSparkJobDisplayName.yaml"
+  controllerClusterId:
+    $ref: "controllerClusterId.yaml"
+  clusterId:
+    $ref: "oceanSparkClusterId.yaml"
+  accountId:
+    $ref: "accountId.yaml"
+  organizationId:
+    $ref: "organizationId.yaml"
+  createdAt:
+    # I had to write createdAt and updatedAt manually and not use createdAtUpdatedAt.yaml.
+    # Otherwise the object is marked as "recursive". This looks like a redoc bug.
+    $ref: "createdAt.yaml"
+  updatedAt:
+    $ref: "updatedAt.yaml"
```

Comment:
@crezvoy why are those two fields not required in the data class in the Python repository? I believe these two fields are always present. Let me know if this is true, I'll update this PR

Reply:
I don't know but I assume it's because it is used as an input parameter for inserting into the DB in the collector.

Reply:
Okay I understand. Thanks!
I'll mark them as required in the API docs though since in this class I only describe responses and not request bodies